app/views/application/_navbar.html.erb

```diff
@@ -7,6 +7,8 @@
     <%= menu_item Match.model_name.human(:count => :many).titleize, matches_path %>
     <%= menu_item Event.model_name.human(:count => :many).titleize, events_path %>
     <%= menu_divider %>
+        <%= menu_item "Create Event", createEvent_path %>
```

Comment:
Please use I18N standards for internationlization and don't hardcode strings. @erksch can you give a short tutorial maybe?

Reply:
Please write your translations in a config/locales/de.yml like
```
de: 
  events:
    create_link: Veranstaltung erstellen 
```
If it does not exist, please create it.
In your view, you can access the translation with:
``<%= t 'events.create_link' %>``
or in your case:
``<%= menu_item t('events.create_link'), createEvent_path %>``

see https://phraseapp.com/blog/posts/rails-i18n-guide/ for more examples :)

Reply:
@erksch Is there a way to also use that method in
<%= link_to '<button type="button">New league</button>'.html_safe, new_event_path %>
to internationalize "New league" ?